edxval/api.py

```diff
@@ -182,6 +143,52 @@ def update_video_status(edx_video_id, status):
     video.save()
 
 
+def get_course_video_image_url(course_id=None, edx_video_id=None, video_image=None):
```

Comment:
We're calling `video_image` after `VideoImage.create_or_update` so why there would be None in video_image ?